src/requestor/routes/maintain.py

```diff
@@ -97,6 +100,29 @@ async def create_request(
         logger.debug(f"Got username from token: {token_username}")
         data["username"] = token_username
 
+    if "revoke" in api_request.query_params:
+        if api_request.query_params["revoke"]:
+            raise HTTPException(
+                HTTP_400_BAD_REQUEST,
+                f"The 'revoke' parameter should not be assigned a value. Received '{api_request.query_params['revoke']}'",
+            )
+        if data.get("resource_path"):
+            # no technical reason for this; it's just not implemented/tested
+            raise HTTPException(
+                HTTP_400_BAD_REQUEST,
+                f"The 'revoke' parameter is not compatible with the 'resource_path' body field",
+            )
+        data["revoke"] = True
```

Comment:
Where exactly are we revoking the policy?

Reply:
in the update endpoint